pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -83,19 +103,70 @@ func TestDefaultConfigMap(t *testing.T) {
 		want *corev1.ConfigMap
 	}{
 		{
-			name: "Test generating default configmap without managed config override",
+			name: "Test generating default configmap without managed config or operator config as overrides",
+			args: args{
+				operatorConfig: &operatorv1.Console{},
+				consoleConfig: &configv1.Console{
+					Spec:   configv1.ConsoleSpec{},
+					Status: configv1.ConsoleStatus{},
+				},
+				managedConfig: &corev1.ConfigMap{},
+				infrastructureConfig: &configv1.Infrastructure{
+					Status: configv1.InfrastructureStatus{
+						APIServerURL: mockAPIServer,
+					},
+				},
+				rt: &routev1.Route{
+					TypeMeta:   metav1.TypeMeta{},
+					ObjectMeta: metav1.ObjectMeta{},
+					Spec: routev1.RouteSpec{
+						Host: host,
+					},
+					Status: routev1.RouteStatus{},
+				},
+			},
+			want: &corev1.ConfigMap{
+				TypeMeta: metav1.TypeMeta{},
+				ObjectMeta: metav1.ObjectMeta{
+					Name:        api.OpenShiftConsoleConfigMapName,
+					Namespace:   api.OpenShiftConsoleNamespace,
+					Labels:      map[string]string{"app": api.OpenShiftConsoleName},
+					Annotations: map[string]string{},
+				},
+				Data: map[string]string{configKey: exampleYaml},
+			},
+		},
+		{
+			name: "Test generating configmap with managed config to override branding but nothing in operator config",
 			args: args{
 				operatorConfig: &operatorv1.Console{
 					TypeMeta:   metav1.TypeMeta{},
 					ObjectMeta: metav1.ObjectMeta{},
-					Spec:       operatorv1.ConsoleSpec{},
-					Status:     operatorv1.ConsoleStatus{},
+					Spec: operatorv1.ConsoleSpec{
```

Comment:
trim?

```go
operatorConfig: &operatorv1.Console{ }
```
I don't believe any of the extra structure is necessary if there are no values set.